utilities/ifix-migration-toolkit/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.4/apache-maven-3.8.4-bin.zip
```

Comment:
Delete `.mvn` directory

Reply:
minor change, to be taken up as part of code cleanup.